Zero km failure1

jacewski-bosch opened the pull request:
## Description
The data which will be provided via this aspect model from an OEM supports the Tier 1 about quality issues with components during assembly at the OEM. It helps to speed up and clarify the root cause of quality problems and to identify affected production batches to narrow NOK goods and to inform the OEM of already delivered packages.


Closes #922 

<!-- The MS2 and MS3 criteria are intended for merges to the main-branch. For small bug-fixes or during the model development, for instance, when merging to a feature branch, you may decide to not fill out the checklists. However, we recommend to follow the MS2 checklist during the development. The MS3 checklist becomes relevant for merges to the main-branch. -->
## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.11.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] payload names and property identifiers must not contain two consecutive underscores ('__') at any position (e.g. `my__model` is not allowed)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.1.0/datatypes.html) have an example value
- [x] all external / imported models have the state "release"
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

jacewski-bosch committed:
Add Zero Kilometer Failure model definition

This file defines the Zero Kilometer Failure model, including properties and characteristics related to part information, vehicle, plant information, and defects.

jacewski-bosch committed:
Add metadata.json with release status

jacewski-bosch committed:
Add RELEASE_NOTES.md for versioning

Document initial model release and changelog.

jacewski-bosch committed:
Refactor prefixes and properties in ZeroKmFailure.ttl

Updated prefixes and properties in ZeroKmFailure.ttl to improve structure and clarity.

jSchuetz88 reviewed (changes requested):
Hi @jacewski-bosch,

thank you for the update. Only some general findings:

- [ ] Many elements don't have the `samm:description` attribute
- [ ] Please use lower case for the `samm:preferredName` attribute

jacewski-bosch committed:
Enhance descriptions and fix VehicleEntity typo

Added descriptions for various characteristics and entities related to quality issues, corrected a typo in 'VehicleEntity', and updated preferred names for clarity.

jacewski-bosch committed:
Merge branch 'eclipse-tractusx:main' into ZeroKmFailure1

jacewski-bosch committed:
Merge branch 'eclipse-tractusx:main' into ZeroKmFailure1

jacewski-bosch committed:
Update quantity dependency to 3.0.0 in ZeroKmFailure.ttl

jSchuetz88 reviewed (approved):
Hi @jacewski-bosch thank you. Works perfectly. LGTM.

jSchuetz88 reviewed (changes requested):
Sorry, I was too hasty. The model looks good to me. But please:

- [ ] update the changelog accordingly
- [ ] plus, the current best practice requires the value of `samm:preferredName` to be in lower case. This will not block your PR, but would be appreciated.

Kind regards,

jacewski-bosch committed:
Change preferred names to lowercase for consistency

jacewski-bosch committed:
Update release version to R26.03

jacewski-bosch committed:
Update io.catenax.zero_km_failure/1.0.0/ZeroKmFailure.ttl

Co-authored-by: Johann Schütz <147107265+jSchuetz88@users.noreply.github.com>

jSchuetz88 committed:
Clean up ZeroKmFailure.ttl by removing blank lines

Removed unnecessary blank lines from the TTL file.

jSchuetz88 committed:
Update dependency to industry_core.common

jSchuetz88 committed:
Format Changelog

jSchuetz88 reviewed (approved):
Thank you for including all changes. Good to go from my side.
